airflow-core/src/airflow/ui/src/components/Banner/BackfillBanner.tsx

```diff
@@ -48,10 +50,17 @@ const buttonProps = {
 
 const BackfillBanner = ({ dagId }: Props) => {
   const { t: translate } = useTranslation("components");
+  const refetchInterval = useAutoRefresh({ dagId });
+
   const { data, isLoading } = useBackfillServiceListBackfillsUi({
     dagId,
+  }, undefined, {
+    refetchInterval: (query: any) =>
```

Comment:
```suggestion
    refetchInterval: (query) =>
```

You don't need any here, typescript can infer the type. 

That will most likley remove the need for manual typing bellow too.